lua/autorun/server/sv_hideholos.lua

```diff
@@ -1,15 +1,29 @@
+util.AddNetworkString( "update_hologram_visibility_preference" )
+util.AddNetworkString( "get_user_hologram_visibility_preference" )
+
 -- Updates the visibility of all holos for a specific player
 local function updateHoloVisibility( ply )
     for _, holo in pairs( ents.FindByClass( "gmod_wire_hologram" ) ) do
         holo:SetPreventTransmit( ply, ply.cfc_holosDisabled )
     end
+
+    net.Start( "update_hologram_visibility_preference" )
+        net.WriteBool( ply.cfc_holosDisabled )
+    net.Send( ply )
 end
```

Comment:
I think these should be split into two separate functions. One to update the persistently-stored preference, and one to do the `SetPreventTransmit` call - then have a single function that calls each separately.

Reply:
I don't think I follow what you're saying.